python/plugins/processing/modeler/ModelerParametersDialog.py

```diff
@@ -360,9 +360,10 @@ def getWidgetFromParameter(self, param):
             else:
                 item = QComboBox()
                 item.setEditable(True)
+                item.addItem(unicode(param.default or "")+" (the default)", param.default)
```

Comment:
The " (the default)" is there to let the user know that this option is the default. If the default is '' or None then we may want to add an option like (the default is empty), otherwise the default option may look a bit strage.